package.json

```diff
@@ -209,15 +209,15 @@
     "react-native-mail": "4.1.0",
     "react-native-markdown-display": "7.0.0-alpha.2",
     "react-native-minimizer": "brunobar79/react-native-minimizer#26ae6dce10d353de701cebc3a1ae126729bf87f0",
-    "react-native-mmkv": "2.3.3",
+    "react-native-mmkv": "2.4.3",
     "react-native-os": "brunobar79/react-native-os#41a08e45260b8c1c53c77ad00e13ae954cb9337c",
     "react-native-pager-view": "5.4.24",
     "react-native-palette-full": "1.2.0",
     "react-native-permissions": "3.0.5",
     "react-native-quick-md5": "3.0.3",
     "react-native-radial-gradient": "rainbow-me/react-native-radial-gradient#1266cd8f6ac3321bcf27357f8771bd088f9f27ef",
     "react-native-randombytes": "3.5.3",
-    "react-native-reanimated": "2.8.0",
+    "react-native-reanimated": "2.9.1",
```

Comment:
why this?

Reply:
the same here, support for RN 69+ added from [2.9.0](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.9.0)

Reply:
I guess it should be [2.9.1](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.9.1) now